.gitignore

```diff
@@ -7,3 +7,7 @@ npm-debug.log
 
 # Mac OS X
 .DS_Store
+
+!intl/
+intl/*
+!intl/en/
```

Comment:
>Let's not bikeshed here.
Sure!

But basically I did not mean alphabetical order; I mean order based on the categories
```
...
# Node.js
...
# Mac OS X
...
```